.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: go
+
+go:
+  - 1.7
+  - 1.8
+  - tip
+
+sudo: required
```

Comment:
@tiago4orion without executing with sudo I was getting "permission denied" on nodejs install. To be honest I just wanted to get over with...was already losing too much time. I read the docs and I'm aware of what you said, the docs also says that you dont need sudo...but it simply did not worked. If you are willing to try to make it work without sudo I would find it much better than the current solution :+1: